docs/docs.styles.scss

```diff
@@ -667,17 +667,17 @@ div[class^="rsg--toolbar"] {
 // Hide private components
 .hide-private,
 .hide-private + div,
-a[href="#private-components"],
-a[href="#private-components"] + ul {
+a[href="/#/Private%20Components"],
+a[href="/#/Private%20Components"] + ul {
```

Comment:
Was it by accident to change this and the line above? Old version was an anchor on `#private-components`, now it's an own url.

Reply:
It's on purpose. As soon as we take care of the parent Private components item in the sidebar we do not need to care about any subitem which would be an anchor. Also, there is not such an anchor at all so I am not sure if it wasn't a bug or maybe it was different in an earlier version of styleguidist.

Reply:
Cool, thanks for clarifying!